python/fledge/services/core/api/audit.py

```diff
@@ -114,14 +116,22 @@ async def create_audit_entry(request):
                    'severity': severity,
                    'details': details
                    }
-        return web.json_response(message)
     except AttributeError as e:
         # Return error for wrong severity method
         err_msg = "severity type {} is not supported".format(severity)
         _logger.error("Error in create_audit_entry(): %s | %s", err_msg, str(e))
-        raise web.HTTPNotFound(reason=err_msg)
+        raise web.HTTPNotFound(reason=err_msg, body=json.dumps({"error": err_msg}))
```

Comment:
we have used "message" KV pair in our API for such scenario. So this needs to be changed.

Reply:
OK